src/main/java/eu/europeana/metis/sandbox/service/workflow/TransformationServiceImpl.java

```diff
@@ -41,12 +51,44 @@ public RecordInfo transform(Record record) {
     return new RecordInfo(Record.from(record, recordTransformed));
   }
 
+  @Override
+  public RecordInfo transformToEdmExternal(Record record) {
```

Comment:
This does not seem to be used anywhere?
That means that the DatasetRepository is not required for this class either?

Reply:
Indeed, it is not used anywhere, but I've decided to keep this method because I believe this will be needed in a future ticket (MET-4083). If you prefer I can remove it, but I believe this could be helpful for the ticket in mind

Reply:
I think it's better to remove this. It's not relevant to the current ticket.
Who knows maybe we won't do the next ticket, or we might do it differently and forget about the code here.
It's simple to re-introduce.

Reply:
Ok then, I will remove this method and DataRepository